src/main/knn/KnnGraphTester.java

```diff
@@ -86,6 +87,7 @@
 import org.apache.lucene.search.IndexSearcher;
 import org.apache.lucene.search.KnnByteVectorQuery;
 import org.apache.lucene.search.KnnFloatVectorQuery;
+import org.apache.lucene.search.OptimisticKnnVectorQuery;
```

Comment:
Hmm bleedover :)  I can remove this after merging or we can fix before ... or, maybe `OptimisticKnnVectorQuery1` gets merged to lucene shortly~

Reply:
removed in second round